indico/modules/events/editing/__init__.py

```diff
@@ -7,7 +7,27 @@
 
 from __future__ import unicode_literals
 
+from indico.core import signals
 from indico.core.logger import Logger
+from indico.modules.events.features.base import EventFeature
+from indico.modules.events.models.events import EventType
+from indico.util.i18n import _
 
 
 logger = Logger.get('events.editing')
+
+
+class EditingFeature(EventFeature):
+    name = 'editing'
+    friendly_name = _('Paper Editing')
+    description = _('Gives event managers the opportunity to let contributors submit papers to be reviewed and '
```

Comment:
`to be edited` ?

Reply:
I thought the same, but is "edited" in that context clear?

Reply:
I thought the same, but is "edited" in that context clear?

Reply:
I'd choose "edited".